ReadMe.md

```diff
@@ -45,4 +45,12 @@ ros2 launch nav2_bringup localization_launch.py map:=./src/warehouse_robot/maps/
 ros2 launch nav2_bringup navigation_launch.py use_sim_time:=true map_subscribe_transient_local:=true
 rviz2 -d ./src/warehouse_robot/config/main.rviz
 
-Install required packages using `install_packages.bash`
\ No newline at end of file
+Install required packages using `install_packages.bash`
+
+### For April Tag Detection
+1. $sudo apt-get install ros-humble-apriltag-ros
```

Comment:
This should be added to the install_packages.bash.  Not as a command here.

Reply:
I'll move it